app/controllers/collaborators_controller.rb

```diff
@@ -0,0 +1,63 @@
+class CollaboratorsController < ApplicationController
+    before_action :authenticate_user!
+    before_action :set_todo_list, only: [:create]
+    before_action :set_collaborator, only: [:destroy]
+  
+    # POST /todo_lists/:todo_list_id/collaborators
+    def create
+        if @todo_list.user != current_user
+            return render json: { error: 'You are not authorized to perform this action' }, status: :forbidden
+        end
```

Comment:
Why are we not using defined policy here? for authorization?

Reply:
forget to remove manual checking , implemented this before integrated controller with policy